recipes/espei/meta.yaml

```diff
@@ -0,0 +1,68 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "espei" %}
+{% set version = "0.3" %}
+{% set sha256 = "cef90367df857468893c6c0949c5e5b4008815bdd8e20be612e94ec82964344d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - six
+    - dask >=0.15
+    - distributed
+    - tinydb >=3
+    - scikit-learn
+    - emcee
+    - pycalphad >=0.5
+    - cerberus
+    - mpi4py
+
+test:
+  imports:
+    - espei
+    - espei.tests
+    - espei.paramselect
+    - espei.core_utils
+    - espei.datasets
+    - espei.plot
+    - espei.run_espei
+    - espei.utils
+
+about:
+  home: http://espei.org
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Fitting thermodynamic models with pycalphad.'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    ESPEI, or Extensible Self-optimizing Phase Equilibria Infrastructure, 
+    is a tool for automated thermodynamic database development within the 
+    CALPHAD method.
+  doc_url: http://espei.org
+  dev_url: https://github.com/PhasesResearchLab/espei
+
+extra:
+  recipe-maintainers:
+    - bocklund
+    - richardotis
```

Comment:
@richardotis -- are you ok being added as a co-maintainer on the conda-forge package for `espei`?

Reply:
I agree to be a co-maintainer of `espei` on conda-forge.